guide/guide/.js/src/main/assets/pages/rest.md

```diff
@@ -608,16 +613,16 @@ Additionally, an implicit instance of `AsRawReal[Raw, Real]` can serve as both.
 
 These implicit instances may come from multiple sources:
 
-* implicit scope of the `Raw` type (e.g. its companion object)
-* implicit scope of the `Real` type (e.g. its companion object)
-* implicits plugged by REST API trait companion
+- implicit scope of the `Raw` type (e.g. its companion object)
+- implicit scope of the `Real` type (e.g. its companion object)
+- implicits plugged by REST API trait companion
   (e.g. `DefaultRestApiCompanion` plugs in `DefaultRestImplicits`)
-* imports
+- imports
 
 Of course, these implicits may also depend on other implicits which effectively means that
 you can use whatever typeclass-based serialization library you want.
 For example, you can define an instance of `AsRaw[JsonValue, Real]` which actually uses
-`Encoder[Real]` from [circe](https://circe.github.io/circe/). 
+`Encoder[Real]` from [circe](https://circe.github.io/circe/).
 See [Customizing serialization](#customizing-serialization) for more details.
 
 ### Serialization implicits summary
```

Comment:
Consider adding similar diagram for streaming or modify existing one (unfortunately I don't see the source anywhere...)